sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/LoginController.java

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class LoginController {
+    @GetMapping("/login")
```

Comment:
Hm... currently the login is even shown when a user is already logged in.
Maybe we should check this here and point to index.html if already logged in?

Reply:
At the moment, a user can go to the login page again and will have to enter his/her/their credentials. However, the login part needs to be improved significantly in the future. As a result, I will keep it how it is right now.